src/integration/java/io/pinecone/integration/dataPlane/UpdateFetchAndQueryPodTest.java

```diff
@@ -91,10 +88,10 @@ public void updateAllParamsFetchAndQuerySyncTest() throws InterruptedException {
             for (String key : upsertIds) {
                 assert (fetchResponse.containsVectors(key));
             }
-        });
+        }, 3);
 
-        String idToUpdate = upsertIds.get(0);
-        List<Float> valuesToUpdate = Arrays.asList(201F, 202F, 203F);
+        String idToUpdate = sparseUpsertIds.get(0);
```

Comment:
Something that @ssmith-pc [pointed out in my latest PR](https://github.com/pinecone-io/pinecone-java-client/pull/115#discussion_r1571581330) is that it's hard to understand quickly when looking at a test when values are like `...get(0)`. It'd be more obvious to readers if this value was just hardcoded. Is it possible to do that here?

Reply:
We could, but then it feels like you're using a hard-coded string that's being generated / defined in a different file (`TestResourcesManager`). Is that still not somewhat confusing as we then have to map the strings used in the tests back to their origin? I might be misunderstanding the feedback.